relay-server/src/services/processor.rs

```diff
@@ -2848,7 +2913,7 @@ mod tests {
             let f = |org_ids: &mut Vec<u64>, msg: Store| {
                 let org_id = match msg {
                     Store::Metrics(x) => x.scoping.organization_id,
-                    Store::Envelope(_) => panic!("received envelope when expecting only metrics"),
+                    _ => panic!("received envelope when expecting only metrics"),
```

Comment:
I'm not sure whether this is an improvement: extending `Store` with new variants results in this message being stale. I'd suggest either not having a catch-all, or updating the panic message to not tie it to "received envelope".

Reply:
That test only expects metrics, anything else should never arrive -> fail the test, I don't want to update the test every time we add more functionality to the processor.